README.md

```diff
@@ -21,7 +21,7 @@
 ## What is TotalCross?
 TotalCross is tools to make easy and fast GUI creation:
  
-* KnowCode:  A computer vision project to help developers to build their screen faster based on image made by design using a library to allow run android XML UI + TotalCross SDK on Linux Arm, iOS, Android and more;
+* [KnowCode](https://github.com/TotalCross/KnowCodeXML):  A computer vision project that aims to help developers build their screens faster based on image made by designers and generate totocross screens ready to run on Android, iOS and Linux ARM devices
```

Comment:
Please, change it to:
* KnowCode: A computer vision project to help developers build user interfaces in less time. A library that converts a design image to a file that runs android XML UI + TotalCross SDK on Linux Arm, iOS, Android and more;

Reply:
done